### DISCUSSION TITLE
white screen flash before screenshot

### DISCUSSION THREAD

**Discussion by chtomek** [flameshot-org/flameshot#3806] [Q&A] [answered]
Is it normal that screen flashes white on screenshot? This is rather disturbing.

[Screencast from 2024-12-31 16-35-07.webm](https://github.com/user-attachments/assets/28fadb94-7ad0-4a6e-aeb9-c1fa85a3d388)

![image](https://github.com/user-attachments/assets/4d138115-1340-43e4-9ee5-66ea625db5ac)

**Comment by mmahmoudian** [comment, marked as answer]
That is GNOME telling you that screenshot is taken. This is part of their "security" feature to let the user know their screen is being captured. Neither Flameshot nor any other independent software can do anything about this. The correct place to complain about this is to GNOME devs:

https://gitlab.gnome.org/GNOME

Imho this is a good feature but the user should be able to toggle it off at least for some whitelisted software.

As far as I can recall, Gnome is the only one forcing such visual effects on users (maybe in their recent version they have added a way to toggle it off in the settings 🤷). But if that disturb you, you can switch your DE and use DEs like KDE Plasma or Budgie or Cinnamon instead of Gnome.

**Reply by chtomek** [reply]
Thank you for the answer, in the mean time i've done more investigation and it confirms what you wrote. I agree it's a good feature and I know Gnome strive to limit amount of settings in the system, but this flash is rather disturbing. Here is Gnome thread about it: [https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3866](https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3866)

**Reply by mmahmoudian** [reply]
I don't use Gnome because of their opinionated design and their devs' attitude towards other devs and users. If the flash is annoying, you can raise your concern to Gnome devs and/or change your DE. In Linux works we have the freedom of choice.
